docs/sitemap.xslt

```diff
@@ -0,0 +1,16 @@
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
```

Comment:
Maybe add a comment about how this is processed/used in the build .. either in this file or in the README of the docs folder

Reply:
Added.